core/domain/platform_parameter_list.py

```diff
@@ -85,8 +86,12 @@ class ParamNames(enum.Enum):
     ' page.',
     platform_parameter_domain.FeatureStages.PROD)
 
-
 Registry.create_feature_flag(
     ParamNames.DIAGNOSTIC_TEST,
     'This flag is for the diagnostic test functionality.',
     platform_parameter_domain.FeatureStages.PROD)
+
+Registry.create_feature_flag(
+    ParamNames.SIGNIN_WITH_POPUP,
+    'This flag is for the new signin with popup feature.',
+    platform_parameter_domain.FeatureStages.PROD)
```

Comment:
@seanlip sorry if I forgot a previous conversation, but are you fine with new features being directly ready for production use?

Reply:
Ah, I missed this, it should actually be in TEST. See instructions here: https://github.com/oppia/oppia/wiki/Developing-new-features-with-feature-gating#how-to-use-feature-flags